<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddingParty</class>
 <widget class="QWidget" name="AddingParty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить мероприятие</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/source/konoha.ico</normaloff>:/source/konoha.ico</iconset>
  </property>
  <widget class="QPushButton" name="acceptParty">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="declineParty">
   <property name="geometry">
    <rect>
     <x>213</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>30</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="placeLineEdit">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>180</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="apperanceLineEdit">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>130</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="partyDateEdit">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>80</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="infoGroupLineEdit">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>230</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="partyNameLabel">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>10</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Название мероприятия</string>
   </property>
  </widget>
  <widget class="QLabel" name="partyDateTimeLabel">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>50</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Дата и время проведения</string>
   </property>
  </widget>
  <widget class="QLabel" name="appearencePartyLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Обязательность явки</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="placePartyLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>159</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Место проведения</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="partyGroupLabel">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>210</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Участвующие группы</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
